sdk/include/psdk/winuser.h

```diff
@@ -1825,6 +1825,19 @@ extern "C" {
 #define WM_MOUSEHOVER               0x02A1
 #define WM_MOUSELEAVE               0x02A3
 
+#define WTS_CONSOLE_CONNECT 0x1
+#define WTS_CONSOLE_DISCONNECT 0x2
+#define WTS_REMOTE_CONNECT 0x3
+#define WTS_REMOTE_DISCONNECT 0x4
+#define WTS_SESSION_LOGON 0x5
+#define WTS_SESSION_LOGOFF 0x6
+#define WTS_SESSION_LOCK 0x7
+#define WTS_SESSION_UNLOCK 0x8
+#define WTS_SESSION_REMOTE_CONTROL 0x9
+#define WTS_SESSION_CREATE 0xa
+#define WTS_SESSION_TERMINATE 0xb
```

Comment:
Nitpick: 0xA and 0xB in uppercase, for consistence.

Reply:
OK.